core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/RawlsBillingProjectComponent.scala

```diff
@@ -577,5 +604,20 @@ trait RawlsBillingProjectComponent {
 
     def setGoogleSyncTime(syncTime: Option[Instant]): WriteAction[Int] =
       query.map(_.googleSyncTime).update(syncTime.map(Timestamp.from))
+
+    def setStatus(status: BillingAccountChangeStatus): WriteAction[Int] =
+      query.map(_.status).update(status.toString)
+
+    def ignoreAllOutstanding(billingProjectName: RawlsBillingProjectName): WriteAction[Int] =
+      query
+        .filter(c =>
+          c.billingProjectName === billingProjectName.value && c.status === BillingAccountChangeStatus.Outstanding.toString
+        )
+        .map(_.status)
+        .update(BillingAccountChangeStatus.Ignored.toString)
+
+    def nextOutstanding(): BillingAccountChangeQuery =
+      query.filter(_.status === BillingAccountChangeStatus.Outstanding.toString).sortBy(_.id).take(1)
```

Comment:
`nextOutstanding()` is the query we'll eventually switch over to for polling, instead of the expensive query we have now. This results in a very light SQL statement:
```sql
select
  `ID`, `BILLING_PROJECT_NAME`, `USER_ID`, `PREVIOUS_BILLING_ACCOUNT`, `NEW_BILLING_ACCOUNT`, `CREATED`, `GOOGLE_SYNC_TIME`, `STATUS`, `OUTCOME`, `MESSAGE`
from `BILLING_ACCOUNT_CHANGES`
where `STATUS` = 'Outstanding'
order by `ID`
limit 1
```